code/index.html

```diff
@@ -1,21 +1,166 @@
 <!DOCTYPE html>
 <html>
-
   <head>
+    <link rel="icon" href="./favicon.ico" style="font-size: large" />
+    <script
+      src="https://kit.fontawesome.com/c38d857281.js"
+      crossorigin="anonymous"
+    ></script>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
+    <title>AT Reinigung</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
-
-      <img src="example-recipe-background.png" alt="Banana Bread">
+    <div class="container2">
+      <div class="logo">
+        <img src="logo2.png" alt="logo" />
+      </div>
+    </div>
+    <nav>
+      <a href="./index.html">Home</a>
+      <a href="#services">Services</a>
+      <a href="#contact">Contact us</a>
+      <a class="appointment-button" href="mailto:info@atreinigung.ch"
+        >Book now</a
+      >
+    </nav>
```

Comment:
Instead of using `position: absolute` for your header, I'd recommend using flexbox. You could wrap the `nav` in the `container2` class and apply flexbox there. It would simplify your html and css (see other comment)
```suggestion
    <div class="container2">
      <img src="logo2.png" alt="logo" />
      <nav>
        <a href="./index.html">Home</a>
        <a href="#services">Services</a>
        <a href="#contact">Contact us</a>
        <a class="appointment-button" href="mailto:info@atreinigung.ch">Book now</a>
      </nav>
    </div>
```